### PR TITLE
[feature5] New feature5 was implemented

### PR DESCRIPTION
**Pull request by tenkarendys**
I have create new folder feature5 with two files.

### CODE REVIEW

File: feature5/new-file-from-karendys.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+New file in the feature5 branch.
```
Comment (nazifullah1) [inline feature5/new-file-from-karendys.txt:1]: It is really good that you added a new file in the feature5 branch.

**Review by nazifullah1 — APPROVED** [review]
Nice jobe, tenkarendys

**Review by nazifullah1 — APPROVED** [review]
Good job!

**Review by MikeMitusov — APPROVED** [review]
I approve!

**Review by bstashchuk — APPROVED** [review]

**Pull request merged**